test/unit_test/mpi_event_listener.cpp

```diff
@@ -55,11 +57,11 @@ class MPIEventListener final : public Catch::EventListenerBase
     if (!test_case_stats.totals.assertions.allOk())
     {
       int test_result = 0;
-      El::mpi::AllReduce(&test_result,
-                         1,
-                         El::mpi::MIN,
-                         El::mpi::COMM_WORLD,
-                         El::SyncInfo<El::Device::CPU>{});
+      h2_tmp::allreduce(&test_result,
+                        1,
+                        Al::ReductionOperator::land,
```

Comment:
so the minimum changed to a logical and? If it's safe to do with integers (and it probably is), I'm fine with it (shouldn't it be an arithmetic and?)

Reply:
Yeah, this is safe. There's even an example with it in the MPI spec (Example 11.13 of the 4.1 spec) with `MPI_INT` as the datatype. Re "arithmetic and", the spec gives you two options: `MPI_LAND` and `MPI_BAND` (aw shucks, I've always wanted to be in a band...), so I'm sticking with the logical :)